private_set_intersection/c/BUILD

```diff
@@ -45,21 +45,21 @@ cc_library(
 )
 
 cc_library(
-    name = "psi_cardinality_client",
-    srcs = ["psi_cardinality_client.cpp"],
-    hdrs = ["psi_cardinality_client.h"],
+    name = "psi_client",
```

Comment:
For JS we have three releases: 

1. Client
2. Server
3. Combined (both client and server implementations. 

This was done in an attempt to reduce size given in a production use-case only one variant would be used. It is easy to do this for JS, but I’m not sure how relevant or feasible it would be for other package managers in different languages.

Reply:
I think the way it's currently set up is fine. When building from source (C++, C, Go), we want as little coupling as possible, and so have separate libraries for client and server. But for Python and JS we will deploy compiled packages, and there the bundle size matters, so combining both client and server into a single package makes sense since they share a lot of code. For JS it's easy to additionally have separate packages to further reduce the bundle size if only client or server is needed, but I don't think this makes sense for Python, because the difference in size will be minimal.